config/environments/development.rb

```diff
@@ -11,6 +11,9 @@
 
   # Show full error reports.
   config.consider_all_requests_local = true
+  
+  # Enable serving static assets
+  config.public_file_server.enable
```

Comment:
@techpeace The docs say that the default is true. So it's not clear to me on why we need to set this for `development.rb`.